manager/README.md

```diff
@@ -8,48 +8,109 @@ The `manager` binary includes all of the controllers that this project defines b
 - `enable-blueprint-controller` to enable the controller for `Blueprint`
 - `enable-motion-controller` to enable the controllers for `BatchTransfer` and `StreamTransfer`
 
+
+## Run and debug locally
+
+Beyond testing, you may run and debug the manager outside the cluster using the following instructions.
+
+### Run required components
+
+Components such as connectors need to be running before you run the manager.
+This can be done by one of these options:
+1. Running components locally directly (no instructions provided)
+2. Running components in a cluster 
+
+For option 2, the Helm installation allows you to pick which compoenents to install. 
+Follow the installation guide as usual but in the Helm installation for the control plane add `--set manager.enabled=false` to skip the deployment of the manager. For example:
+
+```bash
```

Comment:
"kubectl create ns m4d-system" before...

Reply:
btw, m4d-crd should be installed as well right?

Reply:
It says follow the installation instructions as usual. So no need to create namespace (it's created during vault install) and do need to install crds.